### DISCUSSION TITLE
Create project roadmap to help us on developer process

### DISCUSSION THREAD

**Discussion by renaldodev** [diego3g/rsxp-2023#4] [Ideas]
It should be fancy if we get a roadmap predefined and anyone sees the project's current state and the next steps.

**Comment by devmatheusmota** [comment]
Great idea

**Comment by Leo-FSardenberg** [comment]
Dude going full english so that the ppl in the community that can't speak it have no clue of what's being discussed... should add a translation of this one at some point

**Comment by diego3g** [comment]
Vou criar hoje! (nesse repositório vou manter as discussões em português mesmo <3)

Vou fechar essa discussão porque estamos com bastante discussões abertas com tópicos que super valem a contribuição de vocês por lá também e acho que não vale estender muito a discussão por aqui. Sintam-se livres para reabrir a discussão se fizer sentido 🤗
